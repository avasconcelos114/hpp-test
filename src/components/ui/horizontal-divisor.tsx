import React from 'react';

export function HorizontalDivisor() {
  return (
    <hr
      data-testid='horizontal-divisor'
      className='border-grays-line-gray w-full'
      aria-hidden='true'
    />
  );
}
