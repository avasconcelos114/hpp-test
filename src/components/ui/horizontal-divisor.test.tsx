import React from 'react';
import { describe, it, expect } from 'vitest';
import { render, screen } from '@testing-library/react';
import { HorizontalDivisor } from './horizontal-divisor';

describe('HorizontalDivisor', () => {
  it('should render the horizontal divisor', () => {
    render(<HorizontalDivisor />);
    expect(screen.getByTestId('horizontal-divisor')).toMatchSnapshot();
  });
});
