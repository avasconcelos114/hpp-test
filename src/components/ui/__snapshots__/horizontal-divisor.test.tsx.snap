// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HorizontalDivisor > should render the horizontal divisor 1`] = `
<hr
  aria-hidden="true"
  class="border-grays-line-gray w-full"
  data-testid="horizontal-divisor"
/>
`;
